pkg/usage/usageiface/servicer.go

```diff
@@ -0,0 +1,23 @@
+//
+
+package usageiface
+
+import (
+	"time"
+
+	"github.com/Optum/dce/pkg/usage"
+)
+
+// Servicer ...
+type Servicer interface {
+	// UpsertLeaseUsage creates a new lease usage record
+	UpsertLeaseUsage(data *usage.Lease) error
+	// GetLease gets a lease usage record
+	GetLease(id string) (*usage.Lease, error)
+	// GetLease gets a lease usage record
+	GetPrincipal(id string, date time.Time) (*usage.Principal, error)
+	// ListLease returns usage lease usage records
+	ListLease(query *usage.Lease) (*usage.Leases, error)
```

Comment:
Short names are good, but when they come at the expense of losing meaning I think they become problematic. Maybe this is the Java developer in me, but I think the comment says it all: `ListLease returns usage lease usage records` so maybe the method should be called `ListLeaseUsageRecords`, and the resulting data type should be `usage.LeaseUsageRecords`, etc.